extensions-contrib/parquet-extensions/src/main/java/io/druid/data/input/parquet/ParquetHadoopInputRowParser.java

```diff
@@ -37,14 +37,17 @@
 public class ParquetHadoopInputRowParser implements InputRowParser<GenericRecord>
 {
   private final ParseSpec parseSpec;
+  private final boolean binaryAsString;
   private final List<String> dimensions;
 
   @JsonCreator
   public ParquetHadoopInputRowParser(
-      @JsonProperty("parseSpec") ParseSpec parseSpec
+      @JsonProperty("parseSpec") ParseSpec parseSpec,
+      @JsonProperty("binaryAsString") Boolean binaryAsString
```

Comment:
if you use the primitive boolean here it should default to false if it is not provided